signature/signature.go

```diff
@@ -15,7 +15,8 @@ import (
 )
 
 const (
-	signatureType = "atomic container signature"
+	signatureType       = "atomic container signature"
+	cosignSignatureType = "cosign container image signature"
```

Comment:
Very tentative: Maybe implement the two formats in separate files with a similar structure?

I’m not at all sure how much reuse (or a shared interface?) makes sense.